### PR TITLE
Add feed response statistics

### PR DESCRIPTION
**Pull request by kushagraThapar**
Added feed response statistics (mainly request charge)

### CODE REVIEW

**Comment by Incarnation-p-lee** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/microsoft/azure/spring/data/cosmosdb/common/CosmosdbUtils.java  1
         

Clones added
============
- src/test/java/com/microsoft/azure/spring/data/cosmosdb/core/CosmosTemplateIT.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Microsoft/spring-data-cosmosdb/pullRequest?prid=4748905&bid=15844152)

File: src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/ResponseDiagnostics.java [hunk 1]
```diff
@@ -29,10 +44,41 @@ public String toString() {
...
+                       .append(cosmosResponseStatistics)
+                       .append("}");
+        }
         return diagnostics.toString();
     }
+
+    @Getter
+    public static class CosmosResponseStatistics {
+
+        private double requestCharge;
```
Comment (mbhaskar) [inline src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/ResponseDiagnostics.java]: This and the below variable could be final

File: src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/ResponseDiagnostics.java [hunk 2]
```diff
@@ -29,10 +44,41 @@ public String toString() {
...
+        private String activityId;
```
Comment (mbhaskar) [inline src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/ResponseDiagnostics.java]: These could be final

Comment (kushagraThapar) [reply src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/ResponseDiagnostics.java]: Good point, I will make them final.

**Comment by Incarnation-p-lee** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/com/microsoft/azure/spring/data/cosmosdb/common/CosmosdbUtils.java  1
         

Clones added
============
- src/test/java/com/microsoft/azure/spring/data/cosmosdb/core/CosmosTemplateIT.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Microsoft/spring-data-cosmosdb/pullRequest?prid=4748905&bid=15844152)

**Pull request merged**